### DISCUSSION TITLE
How to get rid of switch tenant popup?

### DISCUSSION THREAD

**Discussion by imabduldev** [abpframework/abp#7945] [Q&A]
How can I get rid of switch tenant popup modal on login screen. Can I set unique username and email to all the tenants and host so, I can login without that popup modal by get the tenant id from IdentityUser nullable tenant key. If identity user have they key then get the tenant and switch it or otherwise if it is null then obviously it is the host user.

Thanks!

**Comment by maliming** [comment]
Related issue https://github.com/abpframework/abp/issues/7903

**Reply by imabduldev** [reply]
Can I do this https://docs.aspnetzero.com/en/aspnet-core-angular/latest/Core-Angular-Sign-In-Without-Specifying-Tenant
in abp.io?

**Reply by maliming** [reply]
Yes, They are similar.
